frontend/src/components/maps/leaflet/InfoSlideOut/InfoSlideOut.tsx

```diff
@@ -98,13 +107,72 @@ export type InfoControlProps = {
  */
 const InfoControl: React.FC<InfoControlProps> = ({ open, setOpen, onHeaderActionClick }) => {
   const popUpContext = React.useContext(PropertyPopUpContext);
+  const leaflet = useLeaflet();
+  const propertyInfo = popUpContext.propertyInfo;
+  const location = useLocation();
+  const history = useHistory();
+  const dispatch = useDispatch();
+  const { getParcel } = useApi();
+  const jumpToView = () =>
+    leaflet.map?.setView(
+      [propertyInfo?.latitude as number, propertyInfo?.longitude as number],
+      MAX_ZOOM,
+    );
+  const zoomToView = () =>
+    leaflet.map?.flyTo(
+      [propertyInfo?.latitude as number, propertyInfo?.longitude as number],
+      MAX_ZOOM,
+    );
 
   useEffect(() => {
     const elem = L.DomUtil.get('infoContainer');
     if (elem) {
       L.DomEvent.on(elem!, 'mousewheel', L.DomEvent.stopPropagation);
     }
   });
+  const addAssociatedBuildingLink = (
+    <>
+      <FaPlusSquare color="#1a5a96" className="mr-1" />
+      <Link
+        to={{
+          pathname: `/mapview`,
+          search: queryString.stringify({
+            ...queryString.parse(location.search),
+          }),
+        }}
+        onClick={() => {
+          toast.dark('Creating new associated building...');
+          getParcel(propertyInfo?.id as number)
+            .then((parcel: IParcel) => {
```

Comment:
we need the parcel in the inventory format with all metadata so that the update doesn't trigger unintended changes.